promotions/bin/importmap

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env ruby
```

Comment:
Investigate if this can go, it's probably here to run the gem standalone

Reply:
Done